src/devices/Common/Iot/Device/Common/NumberHelper.cs

```diff
@@ -9,7 +9,7 @@ namespace Iot.Device.Common
     /// <summary>
     /// Helpers for number.
     /// </summary>
-    internal static class NumberHelper
+    public static class NumberHelper
```

Comment:
I think we should try to keep this internal if possible, since the name is something very generic and might clash with people that are trying to use similar types in their apps. Perhaps keep it as internal, and whoever requires it could still compile the file directly instead of referencing the project.

Reply:
I tried going for consistency, especially since it's possible for some projects to use both.